### PR TITLE
Remove ClassOrganization>>allProtocolNames

### PR DESCRIPTION
**Pull request by jecisc**
#allProtocolNames return the name of the protocols + #'--all--', but this #'--all--' is never used. 

This removes #allProtocolNames and adds #protocolNames instead that does not deal with the instance of AllProtocol. 

The end goal is to remove AllProtocol from that system because this is not really used and makes the MM too complicated for what it does.

### CODE REVIEW

File: src/Deprecated12/ClassOrganization.extension.st [hunk 1]
```diff
@@ -10,8 +10,8 @@ ClassOrganization >> addCategory: aString [
 { #category : #'*Deprecated12' }
 ClassOrganization >> allCategories [
 
-	self deprecated: 'Use #allProtocolsNames instead.' transformWith: '`@rcv allCategories' -> '`@rcv allProtocolsNames'.
-	^ self allProtocolsNames
+	self deprecated: 'Use #protocolsNames instead.' transformWith: '`@rcv allCategories' -> '`@rcv protocolsNames'.
```
Comment (MarcusDenker) [inline src/Deprecated12/ClassOrganization.extension.st]: protocolsNames should be protocolNames, I think

**Review by MarcusDenker — APPROVED** [review]

File: src/Deprecated12/ClassOrganization.extension.st (hunk 1 above)
Comment (jecisc) [reply src/Deprecated12/ClassOrganization.extension.st]: Indeed! I'm fixing that :)

**Review by privat — APPROVED** [review]
I dislike how sometimes you have a random `all` at the beginning of method selectors. Like `allThings` that returns a collection of things. I always have a mental image of https://knowyourmeme.com/memes/all-the-things.

Especially for classes, as I expect implicitly that `things` return the things defined in the class and `allThings` returns things defined in class and in its super-classes. (`allThingsUpTo: nil`)

**Review by privat — APPROVED** [review]

**Comment by jecisc** [comment]
Yes same!
I would have expected the protocols with all superclasses for this method

**Pull request merged**